code/modules/mob/living/carbon/metroid/powers.dm

```diff
@@ -1,4 +1,4 @@
-/mob/living/carbon/slime/proc/Wrap(var/mob/living/M) // This is a proc for the clicks
+/mob/living/carbon/slime/proc/WRAP(var/mob/living/M) // This is a proc for the clicks

```

Comment:
This isn't a define, so it shouldn't be all caps

Reply:
Oops.